develop/toolkit/parachains/quickstart/pop-cli.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Quickstart Parachain Development with Pop CLI
+description: Quickly bootstrap parachain projects, scaffold templates, deploy local networks, and streamline development workflows using Pop CLI.
+---
+
+# Quickstart Parachain Development With Pop CLI
+
+## Introduction
+
+[Pop CLI](https://onpop.io/cli/){target=\_blank} is a powerful command-line tool designed explicitly for rapid parachain development within the Polkadot ecosystem. It addresses essential developer needs by providing streamlined commands to set up development environments, scaffold parachain templates, and manage local blockchain networks.
+
+Pop CLI simplifies parachain development with features like:
+
+- Quick initialization of parachain development environments
+
+- Project scaffolding from predefined parachain templates
+
+- Easy deployment and management of local development networks
+
+Developers can quickly begin coding and testing, significantly reducing setup overhead.
+
+### Install Pop CLI
+
+To install Pop CLI, run the following command:
+
+```bash
+cargo install --force --locked pop-cli
+```
+
+Confirm that Pop CLI is installed by running `pop --help` in your terminal:
+
+```bash
+pop --help
+```
+
+### Set Up Your Development Environment
+
+To develop and build Polkadot SDK-based it's essential to prepare your local environment with the necessary tools and dependencies. The [Install Polkadot SDK Dependencies](/develop/parachains/install-polkadot-sdk/){target=\_blank} guide walks you through this setup step-by-step.
```

Comment:
```suggestion
To develop and build Polkadot SDK-based chains, preparing your local environment with the necessary tools and dependencies is essential. The [Install Polkadot SDK Dependencies](/develop/parachains/install-polkadot-sdk/){target=\_blank} guide walks you through this setup step-by-step.
```

Reply:
Fixed in https://github.com/polkadot-developers/polkadot-docs/pull/538/commits/cf6091a8a444f08447db2bdaa88438254130b286